libraries/rib-base/src/main/java/com/badoo/ribs/routing/state/action/single/ActivateAction.kt

```diff
@@ -72,6 +72,9 @@ internal class ActivateAction<C : Parcelable>(
     private fun prepareTransition() {
         // TODO Consider doing this closer to Router (e.g. result of RoutingActivator.activate)
         transitionElements = item.nodes.flatMap { node ->
+            /** At this point view should've been created as a consequence of [RoutingActivator.activate] */
+            if (!node.isViewless) requireNotNull(node.view)
```

Comment:
Can you provide meaningful error message for every `requireNotNull`? Like "At this point view should've been created by RoutingActivator but it was not". In case if this error appears, it will be more clear, that it is the framework error and not client code one. It is better than some random "node.view is null :(" error.